### ISSUE TITLE
Dark mode issue

### ISSUE THREAD

**Issue by AutomateMyLifee** [dslul/openboard#202]
When using the Android developer setting to force dark mode, the keyboard colors are inverted and unusable.

**Comment by Exilion666** [comment]
Problem still presists, with developer force dark mode it is inverted and not usable

**Comment by ibLeDy** [comment]
I have the same issue, the only ones that work as expected (with the "Override force-dark" developer feature on) are the dark material ones.

Here are a couple of screenshots, for context:

<details>
	<summary>
		Selected theme
	</summary>

![Screenshot_20210507-222043](https://user-images.githubusercontent.com/45886240/117505461-5ade6400-af84-11eb-8cf0-c952bc681ac7.png)
</details>

<details>
	<summary>
		Keyboard appearance
	</summary>

![Screenshot_20210507-222059](https://user-images.githubusercontent.com/45886240/117505464-5b76fa80-af84-11eb-8357-9310ed440e34.png)
</details>

<details>
	<summary>
		Pressing and holding a key
	</summary>

![Screenshot_20210507-222545](https://user-images.githubusercontent.com/45886240/117505467-5c0f9100-af84-11eb-9c1d-bcc9a639e438.png)
</details> 

<details>
	<summary>
		"Override force-dark" developer setting
	</summary>

![Screenshot_20210507-222124](https://user-images.githubusercontent.com/45886240/117505466-5c0f9100-af84-11eb-9e0f-bddd81030551.png)
</details>

**Comment by dslul** [comment]
@ibLeDy what version? 1.4.3 or the older one?

**Comment by ibLeDy** [comment]
Oh sorry, I should've given a bit of info regarding versions.

- Device: `Pixel 4a` (Android 11, Build RQ2A.210405.005)
- openboard: `1.4.3`

**Comment by digital-mystik** [comment]
curious about this as well.. I have noticed that it's fine for a little while, but then will eventually invert

**Comment by AutomateMyLifee** [comment]
Yes I still experience this myself on the latest 1.4.3. I noticed it inverts when I turn into landscape orientation or switch user profiles (use work profile app). The only way I found to fix it is to switch keyboards, turn off force dark mode, switch to open board keyboard, and turn on force dark mode again.

**Comment by M4TH1EU** [comment]
Still having the issue on 1.4.4b on A12

**Comment by RAPXT** [comment]
Sadly this issue is still existend on v. 1.4.5 on Android 11.
